.env.local

```diff
@@ -0,0 +1,9 @@
+DB_HOST=localhost
```

Comment:
This file shouldn't be checked in. It may have snuck into your commit before you added it to .gitignore